exchange/huobi/huobi_endpoint.go

```
@@ -45,16 +46,16 @@ func (self *HuobiEndpoint) fillRequest(req *http.Request, signNeeded bool) {
 }
 
 func (self *HuobiEndpoint) GetResponse(
-	method string, req_url string,
+	method string, reqURL string,
 	params map[string]string, signNeeded bool) ([]byte, error) {
 
 	client := &http.Client{
 		Timeout: time.Duration(30 * time.Second),
 	}
-	req_body, _ := json.Marshal(params)
-	req, _ := http.NewRequest(method, req_url, nil)
+	reqBody, _ := json.Marshal(params)
+	req, _ := http.NewRequest(method, reqURL, nil)
```

why the error is ignored here?